scripts/build-dependencies.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+set -e
+
+function findNode() {
+    return $(which npm 1>/dev/null 2>&1)
+}
+
+while [[ $# -gt 0 ]]
+do
+key="$1"
+
+case $key in
+    --no-color)
+    NOCOLOR=true
+    ;;
+    *) # If it wasn't --no-color, we assume it was the path of the root of the project
+    SUPPLIED_APP_FOLDER_PATH=$key
+    ;;
+esac
+shift
+done
+
+MYPATH=$( cd $SUPPLIED_APP_FOLDER_PATH && pwd )
+ROOT=$MYPATH/..
+EXTRA_NPM_ARGS=""
+
+if [ "$MYPATH" = "" ] ; then
```

Comment:
This is never going to be true because of line 24. Perhaps change this to check `$SUPPLIED_APP_FOLDER_PATH` and move it up to after line 22 instead?